docs/rfc/rfc-7.adoc

```diff
@@ -221,23 +221,23 @@ corrupted data:
 
 - With 1 malicious member in a signing group, we need 2 attempts of the protocol
 in the worst case (`P = (99 choose 51) / (100 choose 51) = 0.49`).
-In the worst case, it takes approximately 2h 30min to sign 15 inputs.
+In the worst case, it takes approximately 3h 30min to sign 15 inputs.
```

Comment:
What's the exact math behind it (and the values below)?

Reply:
There is a ~7 minutes timeout for a single signing attempt. Each input requires a separate signature.
We have 15 inputs and we need 2 attempts for each input.

`15 * 2 * 7min = 21min = 3h 30min` 

Reply:
Ah ok, I used a different timeout hence obtained different results. Makes sense!